mail_debrand/tests/test_mail_debrand.py

```diff
@@ -23,3 +24,18 @@ def test_paynow_debrand(self):
         self.assertIn('Powered by', self.paynow_arch)
         res = self.env["mail.template"]._debrand_body(self.paynow_arch)
         self.assertNotIn('Powered by', res)
+
+    def test_lang_paynow_debrand(self):
+        with mute_logger("odoo.addons.base.models.ir_translation"):
+            self.env["base.language.install"].create(
+                {"lang": "nl_NL", "overwrite": True}
+            ).lang_install()
+        with mute_logger("odoo.tools.translate"):
+            self.env["base.update.translations"].create({"lang": "nl_NL"}).act_update()
+
+        ctx = dict(lang="nl_NL")
+        paynow_template = self.env.ref("mail.mail_notification_paynow")
+        paynow_arch = paynow_template.with_context(ctx).arch
+        self.assertIn("Aangeboden door", paynow_arch)
```

Comment:
This test depends on a translation decided from whoever translates Odoo/OCA modules so looks to me like it will easily break as soon as someone changes the translation.
Can you add a translation in the test and use that one instead?

Reply:
Changed